pytest/tests/sanity/state_sync_fail.py

```diff
@@ -16,11 +16,14 @@
 from cluster import init_cluster, spin_up_node, load_config, get_binary_protocol_version
 from configured_logger import logger
 import requests
+import tempfile
 import utils
 
 EPOCH_LENGTH = 10
 START_AT_BLOCK = int(EPOCH_LENGTH * 2.5)
 
+state_parts_dir = str(pathlib.Path(tempfile.gettempdir()) / 'state_parts')
+
 V1_PROTOCOL_VERSION = 48
 V2_PROTOCOL_VERSION = 135
 
```

Comment:
not sure what is the desired test scenario here but 135 nightly contains the v2 nightshade and this test will perform the 4->5 shards resharding